package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "quill-magic-url",
-  "version": "1.0.3",
+  "version": "2.0.0",
```

Comment:
I bumped the major package version, because, although small, this is a breaking change. If you want, I can revert that and leave it to you to bump the version as you see appropriate.